src/pivot-table/hooks/use-column-width.ts

```diff
@@ -98,10 +105,11 @@ export default function useColumnWidth(
 
     let sumOfWidths = 0;
 
-    const columnWidths = visibleLeftDimensionInfo.map((qDimensionInfo, index) => {
+    const lastRow = headersData.data[headersData.size.y - 1] as HeaderCell[];
+    const columnWidths = lastRow.map((cell, index) => {
```

Comment:
what if we have this case as header? then the title value on last yellow cell on headers data will be an empty string and the calculations about `fitToContentWidth` will be wrong i guess? 
(I think that might be the reason why we only rely on leftGrid data/visibleLeftDimensionInfo instead of only headers data) or maybe i'm missing something 

<img width="707" alt="Screenshot 2023-10-18 at 13 25 20" src="https://github.com/qlik-oss/sn-pivot-table/assets/29652890/761d655a-3088-4621-8b8b-ed1c05256bf4">

Reply:
as of our discussion, seems like your changes working fine with headers on pseudo dims (in a scenario where there is no title to calculate the width of cell)